src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -108,6 +109,15 @@ public void periodic() {
         Logger.getInstance().processInputs(getName(), data);
     }
 
+    private double softLimit(double speed) {
+        if (data.distanceInches > SOFT_MAX_DISTANCE && speed > 0) {
+            data.setPoint = (speed * 0.1);
+        } else if (data.distanceInches < SOFT_MIN_DISTANCE && speed < 0) {
+            data.setPoint = (speed * 0.1);
+        }
+        return speed;
```

Comment:
Why does this return the original speed?

Reply:
Ben told me to